contrib/babelfishpg_tsql/src/collation.c

```diff
@@ -897,8 +900,8 @@ transform_from_cs_ai_for_likenode(Node *node, OpExpr *op, like_ilike_info_t like
 	return transform_likenode_for_AI(node, op);	
 }
 
-static bool
-supported_AI_collation_for_like(int32_t code_page)
+bool
+supported_collation_for_db_and_like(int32_t code_page)
 {
 	if (code_page == 1250 || code_page == 1252 || code_page == 1257)
 		return true;
```

Comment:
Please add comment here explainig the use

Reply:
I understand why we did this for `like` operator but why are we restricting database level collation to these code pages only ?

Reply:
Let's restrict it for now as we have really thought and testing other languages. If requirements comes up then we can test and add support for more collation